start_pose: [0.4, 0.0, 0.4, 0.70711, 0.70711, 0.0, 0.0]
objects: [coke]
gripping_offset: 0.1
zone_release_offset: 0.15
bucket_release_offset: 0.2
base_frame: base_link
ee_frame: ati_link
